doc/telekasten.txt

```diff
@@ -418,6 +418,18 @@ telekasten.setup({opts})
 
         Default: `telescope-media-files`
 
+                                    *telekasten.settings.follow_link_fallback*
+    follow_link_fallback({url})~
+        A customizable fallback handler for links that can't be processed by Telekasten
+
+        Default: `nil`
```

Comment:
We should maybe document the fact that when it defaults to `nil`, telekasten will simply use `xdg-open` or `open`.